build/docker/Dockerfile

```diff
@@ -63,7 +63,7 @@ COPY . .
 
 RUN dotnet publish /repo/src/Microsoft.Health.Fhir.${FHIR_VERSION}.Web/Microsoft.Health.Fhir.${FHIR_VERSION}.Web.csproj -c Release -o "/build" --no-restore
 
-FROM mcr.microsoft.com/dotnet/aspnet:5.0.2-alpine3.12@sha256:efd03222416e60b50eb86183a6018c698eebcf496dfa427711ebd7d5c50b427f AS runtime
+FROM mcr.microsoft.com/dotnet/core/aspnet:5.0.3-alpine AS runtime
```

Comment:
mcr.microsoft.com/dotnet/aspnet:5.0.3-alpine3.12 seems valid image for runtime

Reply:
Fixed in 1d5be02.